lib/background_worker.rb

```diff
@@ -1,30 +1,42 @@
+require 'background_worker/version'
+require 'background_worker/config'
+require 'background_worker/uid'
+require 'background_worker/base'
+require 'background_worker/persistent_state'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.